guides/source/error_reporting.md

```diff
@@ -33,72 +27,70 @@ rescue SomethingIsBroken => error
 end
 ```
 
-with
+with a consistent interface:
 
 ```rb
 Rails.error.handle(SomethingIsBroken) do
   do_something
 end
 ```
 
-This approach provides several benefits:
+Rails wraps all executions (such as HTTP requests, jobs, and `rails runner` invocations) in the error reporter, so any unhandled errors raised in your app will automatically be reported to your error-reporting service via their subscribers.
+
+This means that third-party error-reporting libraries no longer need to insert a Rack middleware or do any monkey-patching to capture unhandled exceptions. ActiveSupport libraries can also use this to non-intrusively report warnings that would previously have been lost in logs.
+
+Using the Rails' error reporter is not required. All other means of capturing errors still work.
 
-* Application or error reporting libraries don't need to insert a Rack middleware to capture unhandled exceptions from requests anymore.
-* To ActiveSupport-aware libraries, this can be used to report errors to the host application.
-* It decouples application code from error reporting libraries.
-* It reduces boilerplate code for handling exceptions.
-* Error reporting libraries will need less monkey-patches and be less intrusive to applications.
+### Subscribing to the Reporter
 
-### Subscribe To The Reporter
+To use the error reporter, you need a _subscriber_. A subscriber is any object with a `report` method. When an error occurs in your application or is manually reported, the Rails error reporter will call this method with the error object and some options.
 
-An error subscriber is expected to have a `report` method that takes an exception object and a few options.
-For example:
+Some error-reporting libraries, such as [Sentry's](https://github.com/getsentry/sentry-ruby/blob/master/sentry-rails/lib/sentry/rails/error_subscriber.rb) and [Honeybadger's](https://docs.honeybadger.io/lib/ruby/integration-guides/rails-exception-tracking/), automatically register a subscriber for you. Consult your provider's documentation for more details.
+
+You may also create a custom subscriber. For example:
 
 ```rb
-# config/initializers/error_subscriber.rb
```

Comment:
I think it's still better to hint people where to put this if they want to add one themselves. This feature is not widely adopted by error-reporting services so many users need to create the subscriber manually.

Reply:
Yeah. I removed it because I felt it was _too strong_ a hint (ie "it has to be in this file"), but I see your point. I'll restore it.